src/Amazon.Lambda.Tools/LambdaPackager.cs

```diff
@@ -451,7 +451,12 @@ private static bool ValidateDependencies(IToolLogger logger, string targetFramew
         /// <param name="publishLocation"></param>
         private static void FlattenPowerShellRuntimeModules(IToolLogger logger, string publishLocation, string targetFramework)
         {
-            var runtimeModuleDirectory = Path.Combine(publishLocation, "runtimes/win/lib/netcoreapp3.1/Modules");
+            var runtimeModuleDirectory = targetFramework switch
+            {
+                "net6.0" => Path.Combine(publishLocation, "runtimes/unix/lib/net6.0/Modules"),
```

Comment:
We need to copy the runtimes from the `unix` directory instead of `win` to support deployments from Unix based OSes as well.